tool/jt.rb

```diff
@@ -756,9 +765,31 @@ def test_mri(*args)
                       else
                         []
                       end
-
       files_to_run = (include_files - exclude_files)
+      
+    end
+
+    cext_tests = files_to_run.select { |f| f.include?("cext/ruby") }
```

Comment:
I was to handle a single test without `--cext`: 

`jt test mri cext/ruby/num2int/test_num2int.rb`